util/src/main/i18n/templates/match.properties

```diff
@@ -211,4 +212,3 @@ admin.start.unknownState = Match could not be started at this time.
 admin.end.unknownError = Match could not be ended at this time.
 
 admin.setPool.activeCycle = You may not adjust the pool while match is cycling.
-
```

Comment:
Why remove this newline?

Reply:
I think this might have been a result of an IDE or text editor setting, I've readded it since that seems preferred.